### PR TITLE
Avatar & Badge

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
I have done Avatar and Badge. Please review. I have attached the demo below

Avatar

image

Badge

image

### CODE REVIEW

File: css/padding.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    padding-left: 4rem;
+    padding-right: 4rem;
+}
+
+.px-5 {
+    padding-left: 5rem;
+    padding-right: 5rem;
+}
+
+.py-0p5 {
```
Comment (surbhi84) [inline css/padding.css:138]: .

File: css/padding.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+}
+
+.py-4 {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+}
+
+.py-5 {
```
Comment (surbhi84) [inline css/padding.css:165]: suggestion: classnames are too abbreviated can be improved.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
```
Comment (surbhi84) [inline index.html:5]: more meta tags can be included

**Review by surbhi84 — APPROVED** [review]
Added a few comments, rest looks good to me!

File: css/badge.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    background-color: red;
+}
+
+.badge-green {
+    background-color: #3DC94F;
+}
+
+.badge-green-avatar {
+    background-color: #3DC94F;
+}
```
Comment (anupkgurung) [inline css/badge.css:33]: use variables instead of hex code.

**Review by anupkgurung — APPROVED** [review]
please try to format the files before pushing. Rest look good.

**Comment by manojncsoni007** [comment]
Looks good

File: css/badge.css (hunk 4 above)
Comment (NandhiniKrishnamoorthy) [reply css/badge.css:33]: Will do that

**Pull request merged**